message-bus-tester/src/main.ts

```diff
@@ -57,8 +57,9 @@ const init_mq = async () => {
       ...test_event_def
     }
 
-    rabbitmq_mq.publish(event);
-  }, 5000)
+    console.log("send attempt");
+    rabbitmq_mq.publish(event).then(() => console.log("send complete"));
+  }, 500)
```

Comment:
These changes were made here: https://github.com/libero/reviewer/commit/c978656c94bb66f8f5e2b172e2fe7a51b97eab95

Not sure the `console.log` is necessary and if not should be removed. @petereast can help us but maybe someone else can comment?

Reply:
I'd expect that was debug left in by @petereast :grin:

Reply:
Yep, that's debug. As a rule of thumb, `console.*` should be removed, but `logger.*` should be left in. Sometimes you might need to lower the level of the logger statement, e.g. `logger.info` -> `logger.debug` or `logger.debug` -> `logger.trace`.

Reply:
I'll remove.